doc_comments_ai/llm.py

```diff
@@ -48,8 +48,8 @@ def __init__(
             "Add a detailed doc comment to the following {language} method:\n{code}\n"
             "The doc comment should describe what the method does. "
             "{inline_comments} "
-            "Return the method implementaion with the doc comment as a markdown code block. "
-            "Don't include any explanations in your response."
+            "Return the method implementation with the doc comment as a markdown code block. "
+            "Don't include any explanations or missing type signature or code in your response."
```

Comment:
Is `or code` really neccessary? Not sure if this can affect the responses in general, do you get also good result without it?

Reply:
Yeah. My Bad, Fixed it

Reply:
Nice! `or missing type signature or code` this works then with haskell? Without it the llm responses with the missing type signature right?

Reply:
Yeah. that's possible. Do you want me to add this change in this PR?

Reply:
Would be awesome 🙂

Reply:
Hey @aravindgopall,  I jumped in and added the check, wdyt? 🙌